### ISSUE TITLE
SBT `testOnly` commands don't show specific test packages to run

### ISSUE THREAD

**Issue by mecampbellsoup** [bitcoin-s/bitcoin-s#127]
## Expected Behavior

When firing up an SBT console and typing `testOnly` + <TAB>, I expect to see paths to all available test packages.

## Actual Behavior

No individual test classes/packages are displayed.

![image](https://user-images.githubusercontent.com/2043821/37784712-d42f28f2-2dce-11e8-80dc-8abc478acd23.png)

User error here?

**Comment by Christewart** [comment]
This can be closed right? Fixed with #128 ?

**Comment by mecampbellsoup** [comment]
No, separate issue. From [the docs](https://www.scala-sbt.org/1.x/docs/Testing.html#Tab+completion) it sounds like SBT supports tab completion for tests out of the box.

Here's more SBT console output, the last few lines are the result of pressing tab to see (at least I hoped to see) a list of test packages to run:

```
17:54:54 › sbt -Djava.library.path=secp256k1/.libs
[info] Loading global plugins from /Users/mec/.sbt/0.13/plugins
[info] Loading project definition from /Users/mec/Sites/crypto/bitcoin-s-core/project
[info] Set current project to bitcoin-s-core (in build file:/Users/mec/Sites/crypto/bitcoin-s-core/)
> testO
testOnly      testOptions
> testOnly
--
> testOnly
--
> testOnly
--
> testOnly
--
> test:compile
[success] Total time: 7 s, completed Mar 26, 2018 5:55:37 PM
> compile
[success] Total time: 0 s, completed Mar 26, 2018 5:55:39 PM
> testOnly org.bitcoins.core.protocol.transaction
[info] ScalaTest
[info] Run completed in 65 milliseconds.
[info] Total number of tests run: 0
[info] Suites: completed 0, aborted 0
[info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
[info] No tests were executed.
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
[info] No tests to run for test:testOnly
[success] Total time: 1 s, completed Mar 26, 2018 5:55:44 PM
> testOnly
--
> testOnly
--
> testOnly
--
> testOnly
```

**Comment by mecampbellsoup** [comment]
Hm looks like it may be an issue with `0.13.13` SBT version: https://github.com/sbt/sbt/issues/2822

**Comment by mecampbellsoup** [comment]
@Christewart confirmed it's an SBT bug as you can see here:

![image](https://user-images.githubusercontent.com/2043821/37984946-96c29896-31c5-11e8-980c-017abe0e9e74.png)

Will open a quick PR to bump to `0.13.17`, the latest in the `0.x.x` release series.

Or should we just go to the `1.x.x` releases?

**Comment by mecampbellsoup** [comment]
Closed by #132
